### PR TITLE
add language support (Simplified Chinese - initial version)

### PR DESCRIPTION
**Pull request by longqzh**

### CODE REVIEW

**Comment by vkbo** [comment]
Also, please use a unique branch name. I'm having problems using the github cli tool to make a local copy as you commited directly to your `dev` branch, which conflicts with my `dev` branch.

File: nw/dialogs/about.py [hunk 1]
```diff
@@ -181,6 +181,7 @@ def _fillAboutPage(self):
                 ("Français", "Jan Lüdke (jyhelle)"),
                 ("Norsk Bokmål", "Veronica Berglyd Olsen"),
                 ("Português", "Bruno Meneguello"),
+                ("Simplified Chinese", "Qianzhi Long"),
```
Comment (vkbo) [inline nw/dialogs/about.py:184]: The entry "Simplified Chinese" should not be in English, but in the local name and language like the other languages.

**Review by vkbo — CHANGES_REQUESTED** [review]
I've reverted the changes to the files that shouldn't have been changed, but there is also one line in `about.py` that needs fixing before I can merge.

File: nw/dialogs/about.py (hunk 1 above)
Comment (longqzh) [reply nw/dialogs/about.py:184]: Hi, @vkbo

Which way is convenient to you?
- pull vkbo/dev, and change the branch name, and fix
- Or just fix directly in my own dev 

Let me know:)

Comment (vkbo) [reply nw/dialogs/about.py:184]: Just fix directly in your own dev is fine. I managed to work around the name conflict on my end, so the problem is solved for this PR. But please make a new branch next time :)

**Review by vkbo — APPROVED** [review]
Fantastic, thanks a lot for contributing this translation :)

**Pull request merged**